release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -41,17 +41,19 @@ This release adds improvements related to the following components and concepts.
 === {op-system-first}
 
 [id="ocp-4-7-enhanced-disk-provisioning"]
-==== Enhanced disk provisioning for LUKS and RAID devices
+==== Enhanced disk provisioning for LUKS, RAID, and FBA DASD
```

Comment:
Is FBA DASD going to be a known term for the reader?

Reply:
I sometimes have this debate with myself. It looks cleaner with just the acronym, and most users who don't know what it stands for but want to know should be able to check the goog, but then there's the matter of style guide consistency. Since you brought it up, I reconsidered and have decided to spell it out in the paragraph but not here in the heading.